dbt/adapters/snowflake/relation.py

```diff
@@ -45,10 +44,7 @@ def dynamic_table_config_changeset(
         config_change_collection = SnowflakeDynamicTableConfigChangeset()
 
         if new_dynamic_table.target_lag != existing_dynamic_table.target_lag:
-            config_change_collection.target_lag = SnowflakeDynamicTableTargetLagConfigChange(
```

Comment:
I think you need to keep this.

Reply:
yeah re added the dataclass, was originally part of the target_lag.py stuff that we ripped out.